test/test-hrtime.c

```diff
@@ -33,21 +33,23 @@
 
 TEST_IMPL(hrtime) {
   uint64_t a, b, diff;
-  int i = 75;
+  /* 50 iterations * 300 ms max (assert-enforced) = 15 seconds. */
+  int i = 50;
   while (i > 0) {
     a = uv_hrtime();
     uv_sleep(45);
     b = uv_hrtime();
 
     diff = b - a;
 
-    /*  printf("i= %d diff = %llu\n", i, (unsigned long long int) diff); */
+    /* printf("i= %d diff = %llu\n", i, (unsigned long long int) diff); */
 
-    /* The windows Sleep() function has only a resolution of 10-20 ms. Check
-     * that the difference between the two hrtime values is somewhat in the
-     * range we expect it to be. */
-    ASSERT(diff > (uint64_t) 25 * NANOSEC / MILLISEC);
-    ASSERT(diff < (uint64_t) 80 * NANOSEC / MILLISEC);
+    /* Be generous with assertions about the difference between the
+     * two hrtime values.
+     * - The Windows Sleep() function has a resolution of 10-20 ms.
+     * - On MacOSX 12 CI we observed diff values between (48, 195) ms. */
+    ASSERT(diff > (uint64_t) 20 * NANOSEC / MILLISEC);
+    ASSERT(diff < (uint64_t) 300 * NANOSEC / MILLISEC);
```

Comment:
Maybe we should `#ifdef` this on macOS.

Reply:
@cjihrig I can add the ifdefs if you want. Not sure how strong your opinion was. IMO, since this is a test of `uv_hrtime` and we're just hoping the OS sleep + scheduler are reasonably tight, I think it's fine as-is.

Reply:
I don't feel very strongly. It was just a thought.